pkg/cloud/services/compute/instance.go

```diff
@@ -102,30 +100,10 @@ func (s *Service) InstanceCreate(clusterName string, machine *clusterv1.Machine,
 		}
 	}
 
-	// Get all network UUIDs
-	var nets []ServerNetwork
-	if len(openStackMachine.Spec.Networks) > 0 {
-		var err error
```

Comment:
I assume this change will ignore the OpenstackMachine Spec networks and only honor the cluster status network
then what's the reason to keep OpenstackMachine Spec networks?

Reply:
Aha, I understand openStackMachine.Spec.Networks is used the instance is connected to multi networks.  I wonder if there is really useful though.

Reply:
@hidekazuna I think multiple network is valid ...